zaas-package/src/main/resources/bin/start.sh

```diff
@@ -159,10 +165,13 @@ if [ "${ATTLS_ENABLED}" = "true" ]; then
   ZWE_configs_spring_profiles_active="${ZWE_configs_spring_profiles_active}attls"
 fi
 
+internalProtocol="https"
 # Verify discovery service URL in case AT-TLS is enabled, assumes outgoing rules are in place
 ZWE_DISCOVERY_SERVICES_LIST=${ZWE_DISCOVERY_SERVICES_LIST:-"https://${ZWE_haInstance_hostname:-localhost}:${ZWE_components_discovery_port:-7553}/eureka/"}
-if [ "${ATTLS_ENABLED}" = "true" ]; then
+if [ "${ATTLS_CLIENT_ENABLED}" = "true" ]; then
     ZWE_DISCOVERY_SERVICES_LIST=$(echo "${ZWE_DISCOVERY_SERVICES_LIST=}" | sed -e 's|https://|http://|g')
+    internalProtocol=http
+    export ZWE_zOSMF_scheme="http"
```

Comment:
Why is there the `ZWE_zOSMF_scheme`. Shouldn't we use just `internalProtocol`?

Reply:
This is in the static definition file, if we keep it like this, even if they disable client attls for whatever reason, they can still have the z/OSMF outbound having the specific property in zowe.yaml

Reply:
Done